doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md

```diff
@@ -1,5 +1,6 @@
 # Upgrade Notes
 ## 11.0.0
+- [Assets] Refactored and added tests for `Pimcore\Model\Asset::getMetadata` method to include the parameters even when `$name` parameter is not provided. Added native type hints for this method.
```

Comment:
```suggestion
- [Assets] Refactored `Pimcore\Model\Asset::getMetadata` method to allow listing of all metadata entries filtered by a specific language. Prior this version, the language filter was only available when a specific metadata name was defined in the parameters. Added native type hints and related tests.
```
what about something like this?

Reply:
oh wait, i don't make mention of the `$raw` then.. i was simplyifing to language because the strictMatch is "linked" to it anyways